validation-test/stdlib/Slice/Inputs/GenerateSliceTests.py

```diff
@@ -16,19 +16,37 @@
                 ('FullWidth', '[]', '[]'),
                 ('WithPrefix', '[-9999, -9998, -9997]', '[]'),
                 ('WithSuffix', '[]', '[ -9999, -9998, -9997]'),
-                ('WithPrefixAndSuffix', '[-9999, -9998, -9997, -9996, -9995]', '[-9994, -9993, -9992]')
+                ('WithPrefixAndSuffix',
+                 '[-9999, -9998, -9997, -9996, -9995]', '[-9994, -9993, -9992]')
         ]:
-            Base = '%s%s%sCollection' % (base_kind, traversal, 'Mutable' if mutable else '')
+            Base = '%s%s%sCollection' % (
+                base_kind, traversal, 'Mutable' if mutable else '')
             testFilename = WrapperType + '_Of_' + Base + '_' + name + '.swift'
             with open(testFilename + '.gyb', 'w') as testFile:
+                run_directive_1 = 'RUN: rm -rf %t ; mkdir -p %t'
+                run_directive_2 = ('RUN: %S/../../../utils/gyb ' +
+                                   '%s -o %t/{testFilename} ' +
+                                   '-D ' +
+                                   'test_path="%S"').format(
+                                       testFilename=testFilename)
+                run_directive_3 = ('RUN: %S/../../../utils/line-directive ' +
+                                   '%t/{testFilename} -- ' +
+                                   '%target-build-swift %t/{testFilename} ' +
+                                   '-o %t/{testFilename}.a.out').format(
+                                       testFilename=testFilename)
+                run_directive_4 = ('RUN: %S/../../../utils/line-directive ' +
+                                   '%t/{testFilename} -- ' +
+                                   '%target-run ' +
+                                   '%t/{testFilename}.a.out').format(
+                                       testFilename=testFilename)
                 testFile.write("""
-//// Automatically Generated From validation-test/stdlib/Inputs/GenerateSliceTests.py
+//// Automatically Generated From GenerateSliceTests.py
 //////// Do Not Edit Directly!
 // -*- swift -*-
-// RUN: rm -rf %t ; mkdir -p %t
-// RUN: %S/../../../utils/gyb %s -o %t/{testFilename} -D test_path="%S"
-// RUN: %S/../../../utils/line-directive %t/{testFilename} -- %target-build-swift %t/{testFilename} -o %t/{testFilename}.a.out
-// RUN: %S/../../../utils/line-directive %t/{testFilename} -- %target-run %t/{testFilename}.a.out
+// {run_directive_1}
+// {run_directive_2}
+// {run_directive_3}
+// {run_directive_4}
```

Comment:
@gribozavr I totally agree - it is really really ugly :-)

Short of doing an ugly work around I think the best solution would be to simply skip linting of this file using the "# flake8: noqa" directive.